From: intrigeri <intrigeri@boum.org>
Date: Tue, 25 Sep 2018 07:23:18 +0000
Subject: AppArmor: allow Web Content processes to check the update status.

Otherwise restarting after updating Tor Browser is broken.

(cherry picked from commit 1b2e6aae02eeb2a651e8911c0cec3f6e7dcc1a77)
---
 apparmor/torbrowser.Browser.plugin-container | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apparmor/torbrowser.Browser.plugin-container b/apparmor/torbrowser.Browser.plugin-container
index 7ec8a00..a70f4ac 100644
--- a/apparmor/torbrowser.Browser.plugin-container
+++ b/apparmor/torbrowser.Browser.plugin-container
@@ -57,6 +57,7 @@ profile torbrowser_plugin_container {
   owner @{torbrowser_home_dir}/fonts/** r,
   owner @{torbrowser_home_dir}/omni.ja r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/extensions/*.xpi r,
+  owner @{torbrowser_home_dir}/TorBrowser/UpdateInfo/updates/[0-9]*/update.{status,version} r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/startupCache/* r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/tmp/* rw,
   owner @{torbrowser_home_dir}/TorBrowser/Data/fontconfig/fonts.conf r,
